### DISCUSSION TITLE
WEMI to WEMI Relationships Questions: Primary relationships

### DISCUSSION THREAD

**Discussion by tmqdeborah** [crystalyragui/MARC2RDA#460] [General]
Here are questions about mapping WEMI to WEMI 'primary' relationship values that are found in main and added entry fields.

=======================================================
Please do not start a new comment to reply to a question; use REPLY instead =======================================================

The MARC fields that carry ‘primary relationship’ values are:
- main entries (provide an access point for the **primary work embodied in a manifestation**) 
  - 130 (a title access point where the title of the work is different from the title of the manifestation)
  - 100/110/111 + 240 (a name/title access point where the title of the work is different from the title of the manifestation)
  - 100/110/111 + 245 (a name/title access point where the title of the work is the same as the title of the manifestation)
  - 245 (a title access point where the title of the work is the same as the title of the manifestation)

- analytical added entries: 730, 700, 710, 711, and 740 with Indicator 2 = 2 (provide access points for **additional (aggregated) works embodied in a manifestation**)

[Aggregate Markers](https://github.com/uwlib-cams/MARC2RDA/discussions/459)
- If Aggregate Markers identify that a manifestation is an aggregate, then it is expected that it is safe to assume that a main entry provides an access point for the one of the following types of works:
  - for a collection aggregate: an aggregating work
  - for a parallel aggregate: an aggregating work
  - for an augmentation aggregate: an augmented single-part or multi-part work.

- If the absence of Aggregate Markers identifies that a manifestation is not an aggregate, then it is expected that we must assume that a main entry is:
  - for a single-part or multi-part work
  - for a work that is not identifiable as either aggregating or not-aggregating.

- We do not yet know what percentage of records will turn out to have main entries that are:
  - identifiable as aggregating works (many?)
  - identifiable as single-part or multi-part works (most?)
  - not identifiable as either aggregating or not-aggregating (some?)

**For main entries** 
- If a main entry is identifiable as:
  - an **aggregating work**, then the appropriate relationship is: 
    - Manifestation **_work manifested_** Work

  - a **multi-part or single-part work**, then the appropriate relationships are both:
    - Manifestation **_expression manifested_** Expression 
    - Expression **_work expressed_** Work

- If a main entry is **not** identifiable as an aggregating work or a multi-part or single-part work, then the relationship must be defaulted to both:
    - Manifestation **_expression manifested_** Expression 
    - Expression **work expressed** Work
  
    We cannot use the default relationship **_related work of manifestation_** for a main entry because RDA [Minimum description of a manifestation](https://access.rdatoolkit.org/en-US_ala-f4fb013b-b64e-37b1-902c-561d27b1e832/section_bwq_pxb_ydb)requires us to include at least one of either a **_work manifested_** or **_expression manifested_** relationship. 

**For analytical added entries**
- As explained in Discussion #455, transformation coding cannot tell whether a heading in a 7XX field (including Indicator 2 = 2) is for an aggregating work, a multi-part expression/work, or a single-part expression/work. 
  
  Therefore, for all analytical added entry 7XX fields, we must use ‘related work of manifestation’ as our default between the described manifestation and an aggregated work that is embodied in the manifestation: 
  - Manifestation  **_related work of manifestation_** Work
    - If no 505 is present, we can add a “Contents:” note as a ‘note on manifestation’ to clarify the relationship.

Let's discuss this vigorously!

**Comment by tmqdeborah** [comment]
### Question: Do you agree that one function of the main entry fields listed is above provide an access point for the primary work embodied in a manifestation?

**Comment by tmqdeborah** [comment]
### Question: Are there any other main entry sources that I have missed?

**Comment by tmqdeborah** [comment]
### Question: Are analytical added entries ever used for works that are actually parts of a described multi-part work rather than separate works that are aggregated by the plan of an aggregating work? 

We cannot go by the use of a 245$p, because that is only telling us that the title of the manifestation has either a title of a whole work or a collective title of an aggregating work. 

These examples might be describing a multi-part work and two part works:
100 0  $a Hippocrates,
245 10 $a Hippocrates. $n Volume XI, $p Diseases of Women 1-2 /
700 02 $a Hippocrates. $t Diseases of Women I.
700 02 $a Hippocrates. $t Diseases of Women II.

100 1  $a Stromberg, John,
245 10 $a Hurly burly. $p My Josephine
700 12 $a Stromberg, John, $t Hurly burly. $p Popularity

These examples are describing an aggregating work and aggregated works:
100 1  $a Raff, Joachim,
245 10 $a String quartets. $n Vol. 3, $p 1 & 5 /
700 12 $a Raff, Joachim, $t Quartets, $n no. 5, op. 138,
700 12 $a Raff, Joachim, $t Quartets, $n no. 1, op. 77,

245 04 $a The composer in the computer age. $n VI, $p Compositions by Insook Choi, James Phelps, Zack Settl, Frances White.
505 0  $a Hok pwah / Zack Settel (19:12) -- Lit / Insook Choi (18:00) -- Sax houses / James Phelps (9:30) -- Trees / Frances White (25:57).
700 12 $a Settel, Zack, $t Hok pwah.
700 12 $a Choi, Insook. $t Lit.
700 12 $a Phelps, James. $t Sax houses.
700 12 $a White, Frances, $t Trees.

**Comment by tmqdeborah** [comment]
### Question: Is it safe to relate an Agent from a main entry field to a Work?

In MARC records, agents who are not actually responsible for a work (are not creators) are often given in a main entry field, e.g., Performers or composers of music works, or authors of short stories or poems, etc.

I think it might be safe, but can we safely assume that the majority of agents who are provided in a main entry are **at least associated with** the work that is described by the record? 

If not, we will need to default to **_has related [Agent] to manifestation_** instead of **_has related [Agent] to work_**.

See:
- AACR2 rules: 21.1; and special instructions for Collections (21.7), Art works (21.16-17) , Musical works (21.18-22), Sound recordings (21.23)
- Original RDA: 6.27.1
- Official RDA:  access point for work "[Include a value of Agent: authorized access point for agent for an agent who is associated with a work](https://access.rdatoolkit.org/en-US_ala-f1aeca83-e610-3202-8706-bdfeb9975449/div_mcz_spc_vfb)."

**Comment by AdamSchiff** [comment]
Yes I think so, the majority certainly.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
Box 352900
Seattle, WA 98195-2900
aschiff @ uw.edu
________________________________

**Comment by GordonDunsire** [comment]
Some ways of thinking about high-level entity to entity association relationship elements:

RDA provides a relationship element for every pair of RDA entities, including a symmetric element between two instances of the same entity.

There is no broader relationship predicate in RDF. Logically, there may be an anti-property 'is not associated with', but this is not broader than 'is associated with'.

Association is synonymous with linked, in linked open data: instance A is associated with instance B links the sub-graphs of instance A and instance B.

Linked open data is a graph, or net, of nodes of instances connected by relationship properties/predicates. The Open World Assumption implies that Berners-Lee's 'one giant global graph' is fully connected; there are no disconnected sub-graphs.

A shortcut relationship element is a pathway of nodes and connections that only specifies the first and last nodes; it ignores intermediate nodes and connectors.

Since every instance node is connected to every other instance node, via the one giant global graph, it is feasible to think of a high-level association relationship as a shortcut that is independent of the path between two instance nodes. In practice, I can associate any two instance nodes that exist without knowing anything about their characteristics, etc. and without knowing of any intrinsic relationship between them. This is similar to the 'six degrees of separation' game in a much wider context: there is no real need to prove that any two people on Earth have an association in one way or another (via one path or another).

Therefore using a high-level RDA relationship element is never unsafe within the RDA bibliographic universe. The choice of element depends on shortest known pathway between instances, as well as the domain and range of the property matching the types of the instances.

The downside is that the broadest relationship has the highest recall and lowest precision in an information retrieval system. This is somewhat alleviated by shortest path decisions on using the relationship.
